environment.root.yml

```diff
@@ -5,11 +5,11 @@ channels:
   - defaults
 dependencies:
   - graphviz=2.40.1
-  - jinja2=2.11.2
+  - jinja2=3.0
   - networkx=2.5
   - pygments=2.6.1
   - python=3.7.4
   - singularity=3.5.2
```

Comment:
Since you are working on the dependencies:

I don't like this pattern of repeating almost entire file contents. Sooner or later the dependencies in the two files will start to differ. Better to _only_ include Singularity in _this_ version of the file, then change the documentation (in the next PR that deals with the docs anyway) to tell people that if they want/need to install Singularity they should _update_ their environment with `environment.root.yml` rather than having users to choose one of the two files to _create_ the environment in the first place.

Reply:
And note that since Singularity is taken from the `conda-forge` channel, you can remove `bioconda` and `defaults` channels. It will speed up the build and may also avoid compatibility issues (using different channels can lead to problems).

Reply:
`bioconda` and `conda-forge` are both needed to install `snakemake`. Besides that, all the other dependencies can be removed.

Reply:
Also, in the `README.md` it is already stated to use `environment.root.yml` if `singularity` is not installed and the user has root permission.

Reply:
Yes, but we want it to say that the users should install `environment.yml`, in any case, and then _update_ with `environment.root.yml` only if they want to install Singularity as well. In that case, `environment.root.yml` should _only_ contain Singularity, and nothing else.

Reply:
And since `bioconda` is not needed to for Singularity (only `conda-forge`), you can remove `bioconda` from `environment.root.yml`.

Reply:
...and of course you will need to update the `README.md` accordingly to reflect this new procedure (one-step if Singularity is already installed and two-step otherwise).